ui/app/js/services/permissions_service.js

```diff
@@ -102,6 +102,9 @@ angular.module("app").factory('Permissions', function($http, $q, ScheduledChange
       url += "?csrf_token=" + encodeURIComponent(data["csrf_token"]);
       return $http.delete(url, data);
     },
+    getProducts: function(){
+      return $http.get('/api/releases/columns/product');
+    }
```

Comment:
No need to do this - you can just pass the Releases service to the permissions controller and use https://github.com/mozilla/balrog/blob/master/ui/app/js/services/releases_service.js#L18 instead.